bluemira/base/design.py

```diff
@@ -285,6 +292,7 @@ class Design(DesignABC):
     _params: Configuration
     _build_config: Dict[str, BuildConfig]
     _builders: Dict[str, Builder]
+    configuration: Configuration = Configuration
```

Comment:
Should this be one of the following?

```python
configuration: Configuration = Configuration()
```
```python
configuration: Type[Configuration] = Configuration
```

Reply:
Actually, since this is an abstract property, should we be assigning it a value?

Reply:
It should be the second. 

In Design possibly not, but I was try to preserve existing functionality